guides/common/modules/con_managing-dhcp-using-smart-proxies.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Managing_DHCP_Using_Smart_Proxies_{context}"]
+= Managing DHCP Using {SmartProxies}
+
+{SmartProxies} can manage DHCP within their network.
+You can use the DHCP module of {SmartProxy} to query for available IP addresses, add new, and delete existing reservations.
+Note that your {SmartProxy} cannot manage subnet declarations.
+ifdef::foreman-el,foreman-deb,katello[]
+For more information on managing subnet declarations, see https://github.com/theforeman/puppet-dhcp[puppet-dhcp].
+endif::[]
+
+The DHCP module can manage various available DHCP providers:
+
+* `dhcp_infoblox` - Infoblox DHCP server
+* `dhcp_isc` - ISC DHCP server over OMAPI
+* `dhcp_remote_isc` - ISC DHCP server over OMAPI with leases mounted through networking
```

Comment:
This should link to the "Configuring ... with external DHCP" section since that documents it now.

Reply:
Applied.